db/db_impl.cc

```diff
@@ -4069,18 +4079,22 @@ std::vector<Status> DBImpl::MultiGet(
         (read_options.read_tier == kPersistedTier && has_unpersisted_data_);
     bool done = false;
     if (!skip_memtable) {
-      if (super_version->mem->Get(lkey, value, &s, &merge_context)) {
+      if (super_version->mem->Get(lkey, value, &s, &merge_context,
+                                  &range_del_agg,
+                                  read_options.ignore_range_deletions)) {
```

Comment:
yeah but we have to add read_options which isn't any better :p. anyways will make this change as discussed above.